### PR TITLE
Add additional functionality to Color API

### PR DESCRIPTION
**Pull request by dphurley**
Additional functionality includes:

- RGB array or list
- RGBA array or list
- HSL array or list
- HSLA array or list
- 'name' now includes all 147 CSS colors

### CODE REVIEW

**Comment by dphurley** [comment]
Hound doesn't one particular line that is only 7 characters "too long." I believe this line is OK as-is.

File: lib/ffaker/color.rb [hunk 1]
```diff
@@ -12,5 +12,55 @@ def name
...
+    def hsl_list
+      "#{random_hue_value},#{random_percentage_string},#{random_percentage_string}"
+    end
+
+    def hsla_array
+      [random_hue_value, random_percentage_string, random_percentage_string, random_opacity_value]
+    end
+
+    def hsla_list
+      "#{random_hue_value},#{random_percentage_string},#{random_percentage_string},#{random_opacity_value}"
```
Comment (marocchino) [inline lib/ffaker/color.rb]: How about this? `hsla_array.join(",")`

**Comment by marocchino** [comment]
LGTM! If you don't want change [that](https://github.com/ffaker/ffaker/pull/283#discussion_r71313698). Tell me I will just merge this.

**Comment by dphurley** [comment]
Ready to be merged. Thanks!

**Pull request merged**